presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -155,12 +167,12 @@ public long getOutputRows()
 
     public long getWrittenBytes()
     {
-        return writtenBytes;
+        return logicalWrittenBytes;
     }
 
     public long getWrittenRows()
     {
-        return writtenRows;
+        return logicalWrittenRows;
```

Comment:
if you didn't rename the getter, why did you rename the field?
or, the other way around, why don't you rename the getter too?

Reply:
I didn't want to introduce a BC break but thought this made things clearer within the class